chains/evm/contracts/test/pools/TokenPool/TokenPool.setChainRateLimiterConfigs.t.sol

```diff
@@ -26,105 +26,123 @@ contract TokenPool_setChainRateLimiterConfigs is TokenPoolSetup {
     s_tokenPool.applyChainUpdates(new uint64[](0), chainUpdates);
   }
 
-  function testFuzz_SetChainRateLimiterConfigs_Success(uint128 capacity, uint128 rate, uint32 newTime) public {
-    // Cap the lower bound to 4 so 4/2 is still >= 2
```

Comment:
These assumes/bounds are outdated, from back when we didn't allow setting 0/0 rate limits